lib/logic/auth/facebook/facebook_sign_client.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter_facebook_login/flutter_facebook_login.dart';
+
+enum SingAnswer { successfully, cancelled, error, logOut }
+
+class FacebookSignClient {
+
+  FacebookSignClient() {
+    _facebookLogin = FacebookLogin();
+  }
+
+  FacebookLogin _facebookLogin;
+
+  Future<SingAnswer> handleSignOut() async {
+    await _facebookLogin.logOut();
+    return SingAnswer.logOut;
```

Comment:
The logOut state is redundant since is the only possible state at this point and this point is the only one in which this state could have sense.